src/effect.cpp

```diff
@@ -1078,12 +1086,12 @@ void load_effect_type(JsonObject &jo)
     if(jo.has_member("reduced_desc")) {
         JsonArray jsarr = jo.get_array("reduced_desc");
         while (jsarr.has_more()) {
-            new_etype.reduced_desc.push_back(_(jsarr.next_string().c_str()));
+            new_etype.reduced_desc.push_back(jsarr.next_string().c_str());
         }
     } else if (jo.has_member("desc")) {
         JsonArray jsarr = jo.get_array("desc");
         while (jsarr.has_more()) {
-            new_etype.reduced_desc.push_back(_(jsarr.next_string().c_str()));
+            new_etype.reduced_desc.push_back(jsarr.next_string().c_str());
```

Comment:
`reduced_desc` and friends are containers of `std::string`? Yeah, that makes my life easier.